server/schema/Resolvers.ts

```diff
@@ -1,16 +1,29 @@
+/* eslint-disable no-underscore-dangle */
 import { IResolvers } from '@graphql-tools/utils';
 import bcrypt from 'bcrypt';
 import randomString from 'randomstring';
 
 import UserDB from '../models/User';
 import QueryDB from '../models/Query';
 import ProjectDB from '../models/Project';
+import sessions from '../utilities/sessions';
 
 const resolvers: IResolvers = {
```

Comment:
if (context.authenticated)?

Reply:
See line 27 in `index.ts`. We are saving a flag of whether the user is authenticated based on the token. This line checks if the user is authenticated before sending the data.

Reply:
Thanks!